DocumentDataAPI/DocumentDataAPI/Exceptions/RowsAffectedMismatchException.cs

```diff
@@ -7,7 +7,7 @@ namespace DocumentDataAPI.Exceptions;
 [Serializable]
 public class RowsAffectedMismatchException : Exception
 {
-    public RowsAffectedMismatchException() : this("Not all rows were inserted due to an unknown error.")
+    public RowsAffectedMismatchException()
```

Comment:
Is this correct?

Reply:
Filen var navngivet forkert på dev, og derfor har den fjernet beskeden 😠